webapp/src/Service/ImportProblemService.php

```diff
@@ -190,6 +191,62 @@ public function importZippedProblem(
             }
         }
 
+        if ($deleteOldData && $problem->getProbid()) {
+            if (abs($problem->getTimelimit() - $defaultTimelimit) > 0.001) {
```

Comment:
Is there really no better way to check for an already set timelimit?

Reply:
Reply to the comment above but I cant:
> The second half "Only useful for existing problems" can be omitted if we only show the box for existing problems, right?

But we show it on the general import screen, where we don't know if it updates a problem, since we find an existing one by basename of the ZIP. #1496 makes that a bit awkward anyway, so maybe we should drop that feature?

Reply to:
> Is there really no better way to check for an already set timelimit?

Like how? I can't think of any.

Reply:
Why does `$problem->getTimelimit() !== $defaultTimelimit` not work?

Reply:
Rounding might make this 2.0000000001 vs 2